src/intermediate_structures.h

```diff
@@ -97,6 +98,12 @@ namespace PostProcessing {
  */
 enum Status { Untrustworthy, Trustworthy };
 
+/**
+ * @brief Converts the Untrustworthy/Trustworthy status to a string.
+ *
+ */
+std::string stringTrustworthy(Status s);
```

Comment:
Very minor, but could we rename to `stringStatus`? The enum itself is `PostProcessing::Status` and so I think that name would be a bit better.

Reply:
Thanks, no problem I will update this now.